src/sss_client/pam_message.c

```diff
@@ -124,6 +124,7 @@ int pack_message_v3(struct pam_items *pi, size_t *size, uint8_t **buffer)
     len += pi->pam_newauthtok != NULL ?
                 3*sizeof(uint32_t) + pi->pam_newauthtok_size : 0;
     len += 3*sizeof(uint32_t); /* cli_pid */
+    len += 3*sizeof(uint32_t); /* child_pid */
```

Comment:
Hi,

you have to add `if (pi->child_pid > 0)` here as well to make the site check in the end not fail.

bye,
Sumit

Reply:
Done.